### PR TITLE
Feature/config loaders

### PR DESCRIPTION
**Pull request by francisco-solis99**
For this issue:
1. Create the rules for the CSS loaders in webpack.config.js
2. Install the CSS loaders as dev dependencies with `npm install style-loader css-loader --save-dev`
3. Create the rules for the JS-Babel  in webpack.config.js file
4. Install the babel loaders as dev dependencies with `npm install @babel/core @babel/preset-env babel-loader --save-dev`
5. Create the rules for the files (images essentially)  in webpack.config.js file
6. Create the rules for the html-loader  in webpack.config.js file
7. Install the html-loader as dev dependency with `npm install html-loader --save-dev`
8. Create folders for css and assets inside of the src folder
9. Create the initial css file (styles.css) without code and import it into the index.js  

**Test**

https://user-images.githubusercontent.com/27605929/185723089-7d910719-ed50-48f2-853c-7b860f40a2e0.mp4

### CODE REVIEW

File: src/css/styles.css [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (marmelendez) [inline src/css/styles.css]: you could add a style just to have an example and verify that it works, something like
```
body {
    background-color: brown;
}
```

File: src/js/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+import '../css/styles.css'; //import the initial css file
```
Comment (marmelendez) [inline src/js/index.js]: also import an asset file to have an example of how we can use images, we can write it for the logo!
```
import image from "../assets/logo.png"

const imageLogo = document.getElementById('logo')
imageLogo.src = image;
```

File: src/css/styles.css (hunk 1 above)
Comment (francisco-solis99) [reply src/css/styles.css]: Thanks  for your feedback it was added 😀

File: src/js/index.js (hunk 2 above)
Comment (francisco-solis99) [reply src/js/index.js]: Thanks for your feedback it was added 😀

**Pull request merged**